provisions/roles/jenkins/master/tasks/jobs.yml

```diff
@@ -20,13 +20,13 @@
 - name: Push job to jenkins
   shell: jenkins-jobs --ignore-cache update {{ ansible_env.HOME}}/index.yml
 
-- name: Ensure /opt/container-pipeline-service/jenkinsbuilder dir exists on Jenkins master
-  file: dest=/opt/container-pipeline-service/jenkinsbuilder state=directory
+- name: Ensure /opt/cccp-service/jenkinsbuilder dir exists on Jenkins master
+  file: dest=/opt/cccp-service/jenkinsbuilder state=directory
```

Comment:
@navidshaikh please read the commit summary here: https://github.com/CentOS/container-pipeline-service/pull/58/commits/586a002d9dcb4759eb498201bfb875184e355b92